admin/load.php

```diff
@@ -757,58 +791,74 @@ function perflab_plugin_admin_notices() {
 		}
+		$message .= '</ol>';
+	}
 
-		if ( 1 === $modules_count ) {
-			$message  = '<p>';
-			$message .= sprintf(
-				/* translators: Module name */
-				esc_html__( 'Your site is using the "%s" module which will be removed in the future in favor of its equivalent standalone plugin.', 'performance-lab' ),
-				esc_attr( $available_module_names[0] )
-			);
-			$message .= ' ';
-			$message .= esc_html__( 'Please click the following button to install and activate the relevant plugin in favor of the module. This will not impact any of the underlying functionality.', 'performance-lab' );
-			$message .= '</p>';
-		} else {
-			$message  = '<p>';
-			$message .= esc_html__( 'Your site is using modules which will be removed in the future in favor of their equivalent standalone plugins.', 'performance-lab' );
-			$message .= ' ';
-			$message .= esc_html__( 'Please click the following button to install and activate the relevant plugins in favor of the modules. This will not impact any of the underlying functionality.', 'performance-lab' );
-			$message .= '</p>';
-			$message .= '<strong>' . esc_html__( 'Available standalone plugins:', 'performance-lab' ) . '</strong>';
-			$message .= '<ol>';
-			foreach ( $available_module_names as $module_name ) {
-				$message .= sprintf( '<li>%s</li>', esc_html( $module_name ) );
-			}
-			$message .= '</ol>';
-		}
+	?>
+	<div class="notice notice-warning is-dismissible">
+		<?php echo wp_kses_post( $message ); ?>
+		<p class="perflab-button-wrapper">
+			<button type="button" class="button button-primary perflab-install-active-plugin">
+				<?php esc_html_e( 'Migrate legacy modules to standalone plugins', 'performance-lab' ); ?>
+			</button>
+			<span class="dashicons dashicons-update hidden"></span>
+		</p>
+	</div>
+	<?php
+}
 
-		?>
-		<div class="notice notice-warning is-dismissible">
-			<?php echo wp_kses_post( $message ); ?>
-			<p class="perflab-button-wrapper">
-				<button type="button" class="button button-primary perflab-install-active-plugin">
-					<?php esc_html_e( 'Migrate legacy modules to standalone plugins', 'performance-lab' ); ?>
-				</button>
-				<span class="dashicons dashicons-update hidden"></span>
-			</p>
-		</div>
-		<?php
+/**
+ * Returns an array of standalone plugins name which currently active modules.
+ *
+ * @since n.e.x.t
+ *
+ * @return string[]
+ */
+function perflab_get_modules_compatible_with_standalone_plugin() {
+	$active_modules_with_plugins = perflab_get_active_modules_with_standalone_plugins();
+	if ( empty( $active_modules_with_plugins ) ) {
+		return;
+	}
+
+	$module_data            = perflab_get_modules();
+	$available_module_names = array();
+	foreach ( $active_modules_with_plugins as $module_slug ) {
+		if ( isset( $module_data[ $module_slug ] ) && ! perflab_is_standalone_plugin_loaded( $module_slug ) ) {
+			$available_module_names[] = $module_data[ $module_slug ]['name'];
+		}
 	}
+	return $available_module_names;
```

Comment:
See above. Also a small nit-pick, the `$module_slug` here is typically referred to as `$module_dir` elsewhere in the code as it contains both the focus directory and the module directory (the latter of which is the module's slug).
```suggestion
	$available_modules_data = array();
	foreach ( $active_modules_with_plugins as $module_dir ) {
		if ( isset( $module_data[ $module_dir ] ) && ! perflab_is_standalone_plugin_loaded( $module_dir ) ) {
			$available_modules_data[] = $module_data[ $module_dir ];
		}
	}
	return $available_modules_data;
```